andela/ah-frontend-lannister
# #160617649 Bookmark articles for reading later

#### What does this PR do?
- Enables users to bookmark articles to read later
#### Description of Task to be completed?
- Enable authenticated users to bookmark or unbookmark articles
#### How should this be manually tested?
- When you click on an article to view, a sidebar with a bookmark appears. 
- Click on the bookmark icon
- To unbookmark, an article, click on profile > bookmarks
- From the list of bookmarks, click on the remove button below the article you want to unbookmark
#### What are the relevant pivotal tracker stories?
[#160617649 ]

#### Screenshoots
- Single article
![screen shot 2018-11-23 at 11 34 41](https://user-images.githubusercontent.com/1590202/48934110-df9fa980-ef13-11e8-9882-ea0f4c3f61e4.png)



 - Bookmark icon
![screen shot 2018-11-23 at 11 34 57](https://user-images.githubusercontent.com/1590202/48934119-ec240200-ef13-11e8-808c-c459a74cb2a7.png)
